lib/Test/Nginx/Util.pm

```diff
@@ -906,7 +906,7 @@ sub write_config_file ($$) {
     if ($LoadModules) {
         my @modules = map { "load_module $_;" } grep { $_ } split /\s+/, $LoadModules;
         if (@modules) {
-            $main_config .= join " ", @modules;
+            $main_config = join " ", @modules, $main_config;
```

Comment:
Better add a space between these two things to make the resulting nginx.conf look a bit better?

Reply:
Oh, never mind, a space is already used as a delimiter in the `join()` call :)

Reply:
This produces the following output:
```
load_module foo; load_module bar; stream {

    server {
        listen 1985;

   content_by_lua_block { ngx.say(ngx.quote_sql_str('hello\n\r\'"\\')) }

    }
}
```

I would be in favor of having the following output, which makes debugging easier when inspecting the generated `nginx.conf`:
```
load_module foo;
load_module bar;
stream {

    server {
        listen 1985;

   content_by_lua_block { ngx.say(ngx.quote_sql_str('hello\n\r\'"\\')) }

    }
}
```

This simple change is my suggestion:
```suggestion
            $main_config = join "\n", @modules, $main_config;
```